src/Tldr/App/Handler.hs

```diff
@@ -1,39 +1,45 @@
-{-#LANGUAGE RecordWildCards#-}
-{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE BangPatterns    #-}
 
 module Tldr.App.Handler
   ( handleAboutFlag
   , retriveLocale
   , checkLocale
   , englishViewOptions
   , getCheckDirs
-  , initializeTldrPages
   , pageExists
   , getPagePath
   , updateTldrPages
   , handleTldrOpts
   ) where
 
-import Control.Monad (unless)
 import Data.Char (toLower)
 import Data.List (intercalate)
 import Data.Semigroup ((<>))
 import qualified Data.Set as Set
 import Data.Version (showVersion)
+import Data.Time.Clock
+
```

Comment:
Minor comment: For style consistent there doesn't need to be any space between them.

Reply:
Not sure what you mean... I committed something, did you mean this?